### PR TITLE
adi_upate_boot.sh: add options to mkdir and wget commands

### PR DESCRIPTION
**Pull request by AAndrisa**

### CODE REVIEW

File: adi_update_boot.sh [hunk 1]
```diff
@@ -476,9 +476,9 @@ restoring_extra_files()
         exit 1
       fi
       echo "u-boot.img restored."
-      echo -e "\nRestoring extlinux/extlinux.conf (specific to cyclone5 projects starting with 2021_r1 release)..."
+      echo -e "\nRestoring extlinux/extlinux.conf (specific to arria10 projects starting with 2021_r1 release)..."
```
Comment (nunojsa) [inline adi_update_boot.sh:479]: this looks an unrelated change...

File: adi_update_boot.sh [hunk 2]
```diff
@@ -264,7 +264,7 @@ fi
 RPI_BOOT_ARCHIVE_NAME="rpi_latest_boot.tar.gz"
 echo -e "\nStart downloading $RPI_BOOT_ARCHIVE_NAME..."
 rm -rf $RPI_BOOT_ARCHIVE_NAME
-wget -nc $rpi_boot_url
+wget --no-check-certificate -nc $rpi_boot_url
```
Comment (nunojsa) [inline adi_update_boot.sh:267]: the commit message should state why we need this option

**Review by nunojsa — COMMENTED** [review]
couple of nits...

File: adi_update_boot.sh [hunk 3]
```diff
@@ -476,9 +476,9 @@ restoring_extra_files()
         exit 1
       fi
       echo "u-boot.img restored."
-      echo -e "\nRestoring extlinux/extlinux.conf (specific to cyclone5 projects starting with 2021_r1 release)..."
```
Comment (SRaus) [inline adi_update_boot.sh:479]: Actually this is specific to cyclone5, but also to A10SOC starting with Quartus 20.1 Pro version.

File: adi_update_boot.sh (hunk 1 above)
Comment (SRaus) [inline adi_update_boot.sh:479]: As Nuno said, I would add another commit for updating these echo's and also would be great to optimize it a bit: 
- lines 455-465 and 479-489 are doing the same thing: restoring extlinux.conf.
- also lines 442-451 and 466-477 are restoring u-boot (src for c5soc / img for a10soc).
You can create methods and just call them here.

**Review by SRaus — CHANGES_REQUESTED** [review]

File: adi_update_boot.sh [hunk 4]
```diff
@@ -436,56 +436,55 @@ restoring_image()
...
+    echo "Warning! $FILE cannot be restored."
+    echo "You will have to manually copy specific boot files in boot partition root (see boot partition Readme.txt)"
+    exit 1
+  fi
+  echo "$FILE restored."
+}
+
+restoring_extlinux()
+{
+  echo -e "\nRestoring extlinux/extlinux.conf (specific to cyclone5 projects starting with 2021_r1 release)..."
```
Comment (SRaus) [inline adi_update_boot.sh:460]: extlinux.conf file is also specific to Aria10 soc projects, not just cyclone5.

**Review by SRaus — CHANGES_REQUESTED** [review]
There is a typo in commit title, on file name, and one in first line of description.

**Review by SRaus — COMMENTED** [review]
There is a typo in commit title on file name.
Also, you can remove the comment change from lines 479-480 since it is handled in the other commit.

**Review by SRaus — APPROVED** [review]

**Review by ccraluca — APPROVED** [review]

**Pull request merged**